cty/function/stdlib/crypto.go

```diff
@@ -0,0 +1,325 @@
+package stdlib
+
+import (
+	"crypto/md5"
+	"crypto/rsa"
+	"crypto/sha1"
+	"crypto/sha256"
+	"crypto/sha512"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/pem"
+	"fmt"
+	"hash"
+
+	uuidv5 "github.com/google/uuid"
+	uuid "github.com/hashicorp/go-uuid"
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/function"
+	"github.com/zclconf/go-cty/cty/gocty"
+	"golang.org/x/crypto/bcrypt"
+)
+
+var UUIDFunc = function.New(&function.Spec{
```

Comment:
Ah I understand, would it be a good idea to add a seed argument to those functions like a string or a number that would be well documented; so that it generates the same value on each call with the same argument values ?